src/js/component.js

```diff
@@ -1093,9 +1297,12 @@ class Component {
 
   /**
    * Get width or height of computed style
-   * @param  {String} widthOrHeight  'width' or 'height'
-   * @return {Number|Boolean} The bolean false if nothing was set
-   * @method currentDimension
+   *
+   * @param {string} widthOrHeight
+   *        'width' or 'height'
+   *
+   * @return {number|boolean}
+   8         the dimention or false if nothing was set
```

Comment:
8 instead of *.
dimension mispelled.

This doesn't return a boolean. It should default to `0`, I believe.